docs/release_notes.md

```diff
@@ -1,5 +1,97 @@
 #
 
+## 4.3.4
+
+Release Date: 2025-03-05
+
+The iRODS Consortium and RENCI are pleased to announce iRODS 4.3.4.
+
+This release primarily focuses on preparing the ground for the initial release of iRODS 5 by introducing more deprecations, fixing bugs, and cleaning up implementations.
+
+The biggest news of this release is that `imiscsvrinfo` has been updated to report SSL/TLS certificate information, if in use. This is made possible due to enhancements to the `rcGetMiscSvrInfo` API endpoint. Keep in mind that this requires the client to have a proper client-side configuration. 
+
+This release consists of [94 commits from 6 contributors](https://github.com/irods/irods/compare/4.3.3...4.3.4) and [closed 134 issues marked for 4.3.4](https://github.com/irods/irods/issues?q=milestone%3A4.3.4).
+
+The latest binary packages for AlmaLinux8, RockyLinux9, Ubuntu20, Ubuntu22, Ubuntu24, Debian11, and Debian12 are available at <https://packages.irods.org/>.
+
+### Changed
+
+- Improve documentation (#2600, #7622, #7701, #7784, #7952, #8000).
+- Improve testing (#6421, #7412, #7491, #7795, #7990, #8046, #8047, #8183, #8185, #8247).
+- Clean up code (#5800, #6972).
+- Remove unnecessary code - source files, functions, declarations, header includes, etc (#6043, #6234, #6546, #7919, #7942, #8010, #8133, #8201, #8204).
+- Replace use of externals-provided libarchive with distro-provided libarchive (#6250, #7286).
+- Improve CMake (#6319, #6584).
+- Update clang-format / clang-tidy configuration (#6970, #7313, #7751, #7821, #7822).
+- Remove ACLs following removal of user (#7778).
+- Migrate GitHub workflows to Ubuntu 24.04 (#7823).
+- Expand permission levels supported by atomic ACL operations API (#7913).
+- Reduce log noise (#7953).
+- Remove installation of packedRei directory from CMakeLists.txt (#7993).
+- Install msiExecCmd_bin/hello script as a template (#7994).
+- Replace stacktrace with user-friendly error message in `itree` (#8082).
+- Improve GenQuery2 parser's handling of whitespace sequences (#8182).
+- Add json_events.hpp header file to development package (#8200).
+
+### Removed
+
+- Remove CentOS 7 build from GitHub workflows (#7968).
+
+### Deprecated
+
+- Deprecate `ilocate` (#2524).
+- Deprecate `igetwild` (#2525).
+- Deprecate modification of user having rodsgroup type in GeneralAdmin API (#2978).     THIS ISSUE IS REFERENCED IN TWO SECTIONS. IT DOES COVER TWO THINGS.
```

Comment:
Normally, we want to reference an issue number once, but this feels like an exceptional case due to what the issue covers.

Reply:
I get what you're saying, but if I remove one of them, we lose important information. The words for each section are important IMO.

If I were going to leave one in, it would probably be the deprecation message since it could be viewed as - _The Consortium must be providing a better alternative._ The line under the "Added" section doesn't give the reader any reason to think that the old way is possibly deprecated (assuming they even know about the old way).

Reply:
Understood.   I think in the long run ... as long as you have a sense for how you want to hold them... and can remain consistent... it's fine.

Reply:
On that note, `iadmin mkuser` with a `rodsgroup` is also deprecated as of this release.

Reply:
Do you feel there's something we should tweak?

Reply:
Updated to match the suggestion.